docs/EN_US/ECLLanguageReference/ECLR_mods/Basics-Constants.xml

```diff
@@ -124,6 +124,39 @@ MyUnicodeString3 := U'abcd\u00EB'; // becomes 'abcdë'«'</programlisting>
     VARSTRING.</para>
 
     <programlisting>MyQString := Q'ABCD'; // same as: (QSTRING)'ABCD'</programlisting>
+
+    <sect3>
+      <title>Multiline Strings</title>
+
+      <para>A multiline string begins and ends with three single quotes (''').
+      </para>
+
+      <para>Any quotes, tabs, or newlines in between the triple quotes are
+      part of the string. While you can use the \ escape character inside a
+      multiline string, escaping is not necessary. </para>
```

Comment:
I think you do need to escape the \ character though, and \ at the end of a line in a multi line comment removes the end of line (joins the two lines together).  worth clarifying that.
@richardkchapman is that correct?

Reply:
Yes, it seems to be